package.json

```diff
@@ -46,7 +46,7 @@
     "eslint-plugin-react": "^7.19.0",
     "eslint-plugin-react-hooks": "^3.0.0",
     "ganache-cli": "^6.12.2",
-    "hardhat": "^2.9.0",
+    "hardhat": "^2.12.6",
```

Comment:
I needed to update hardhat to fix an issue when forking arbitrum documented[ here ](https://github.com/NomicFoundation/hardhat/issues/2995) and fixed in the latest version of hardhat